### DISCUSSION TITLE
DAI 2023-2024 - Practical work 4 - Massimo Stefani - Loïc Herman - Kevin Farine - Olin Bourquin

### DISCUSSION THREAD

**Discussion by massteff** [heig-vd-dai-course/heig-vd-dai-course#192] [Show and tell]
Our project involves creating a virtual machine manager. [Here!](https://github.com/heig-lherman/dai-pw04)

**Comment by KevinFarine** [comment]
1e59cf4f951eb9360fd4ab69492401b85f24e14f

**Reply by massteff** [reply]
1312d9c3f4e2cece29b0f334c449fe4597c26085 ... Sorry for the little delay
